server/valgrind-out.txt

```diff
@@ -1,114 +1,136 @@
+--4201-- REDIR: 0x4928630 (libc.so.6:strlen) redirected to 0x48311d0 (_vgnU_ifunc_wrapper)
+--4201-- REDIR: 0x4931bb0 (libc.so.6:memrchr) redirected to 0x48311d0 (_vgnU_ifunc_wrapper)
+--4201-- REDIR: 0x4929850 (libc.so.6:strcasecmp_l) redirected to 0x48311d0 (_vgnU_ifunc_wrapper)
+--4201-- REDIR: 0x49293d0 (libc.so.6:memchr) redirected to 0x48311d0 (_vgnU_ifunc_wrapper)
+--4201-- REDIR: 0x4945930 (libc.so.6:wcslen) redirected to 0x48311d0 (_vgnU_ifunc_wrapper)
+--4201-- REDIR: 0x4928ae0 (libc.so.6:strspn) redirected to 0x48311d0 (_vgnU_ifunc_wrapper)
+--4201-- REDIR: 0x4929750 (libc.so.6:stpncpy) redirected to 0x48311d0 (_vgnU_ifunc_wrapper)
+--4201-- REDIR: 0x49296f0 (libc.so.6:stpcpy) redirected to 0x48311d0 (_vgnU_ifunc_wrapper)
+--4201-- REDIR: 0x492ac90 (libc.so.6:strchrnul) redirected to 0x48311d0 (_vgnU_ifunc_wrapper)
+--4201-- REDIR: 0x49298a0 (libc.so.6:strncasecmp_l) redirected to 0x48311d0 (_vgnU_ifunc_wrapper)
+--4201-- REDIR: 0x4a11500 (libc.so.6:__strrchr_avx2) redirected to 0x483ea10 (rindex)
+--4201-- REDIR: 0x4a14b60 (libc.so.6:__memset_avx2_unaligned_erms) redirected to 0x48428e0 (memset)
+--4201-- REDIR: 0x49230e0 (libc.so.6:malloc) redirected to 0x483b780 (malloc)
+--4201-- REDIR: 0x49236d0 (libc.so.6:free) redirected to 0x483c9d0 (free)
+--4201-- REDIR: 0x4928fa0 (libc.so.6:__GI_strstr) redirected to 0x4843ca0 (__strstr_sse2)
+--4201-- REDIR: 0x4a0d530 (libc.so.6:__memchr_avx2) redirected to 0x4840050 (memchr)
+--4201-- REDIR: 0x4a146e0 (libc.so.6:__memcpy_avx_unaligned_erms) redirected to 0x48429f0 (memmove)
+--4201-- REDIR: 0x4a110e0 (libc.so.6:__strchr_avx2) redirected to 0x483ebf0 (index)
+--4201-- REDIR: 0x4923e80 (libc.so.6:realloc) redirected to 0x483df30 (realloc)
+--4201-- REDIR: 0x4a116d0 (libc.so.6:__strlen_avx2) redirected to 0x483ef40 (strlen)
+--4201-- REDIR: 0x4a0dcc0 (libc.so.6:__memcmp_avx2_movbe) redirected to 0x48421e0 (bcmp)
+--4201-- REDIR: 0x4a11310 (libc.so.6:__strchrnul_avx2) redirected to 0x4843540 (strchrnul)
+--4201-- REDIR: 0x4a12c10 (libc.so.6:__strcpy_avx2) redirected to 0x483f090 (strcpy)
+--4201-- REDIR: 0x4924b10 (libc.so.6:calloc) redirected to 0x483dce0 (calloc)
+--4201-- REDIR: 0x4a0cbd0 (libc.so.6:__strcmp_avx2) redirected to 0x483fed0 (strcmp)
+--4201-- REDIR: 0x4a146c0 (libc.so.6:__mempcpy_avx_unaligned_erms) redirected to 0x4843660 (mempcpy)
+--4201-- REDIR: 0x4a12fa0 (libc.so.6:__strncpy_avx2) redirected to 0x483f230 (strncpy)
+==4201== 
+==4201== HEAP SUMMARY:
+==4201==     in use at exit: 8 bytes in 1 blocks
+==4201==   total heap usage: 172 allocs, 171 frees, 356,570 bytes allocated
+==4201== 
+==4201== Searching for pointers to 1 not-freed blocks
+==4201== Checked 90,824 bytes
+==4201== 
+==4201== 8 bytes in 1 blocks are still reachable in loss record 1 of 1
+==4201==    at 0x483B7F3: malloc (in /usr/lib/x86_64-linux-gnu/valgrind/vgpreload_memcheck-amd64-linux.so)
+==4201==    by 0x4860959: timer_create@@GLIBC_2.3.3 (timer_create.c:59)
```

Comment:
@abaa3023 I think this is pointing out a missing `timer_delete()` call in cleanup.